devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -438,9 +437,19 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
         return false;
     }
 
-    if (!(config.check("floatingBaseFrame") && config.find("floatingBaseFrame").isList()
-          && config.find("floatingBaseFrame").asList()->size() == 2)) {
-        yError() << LogPrefix << "floatingBaseFrame option not found or not valid";
+    yarp::os::Bottle* floatingBaseFrameList;
```

Comment:
@lrapetti may be using two parameters as following:

- `baseFrame` that is a list of size two
- `fixedBase` that is a bool, which if set to false will be a floating base configuration

will be a clean and concise code. What do you think ?

Reply:
In both the case we should pass the name of the model base frame, indeed we can not use a bool. 

Anyway, I have noticed we were passing this option as a `List` but we are just using the first element of the list (the name of the base frame in the model). Indeed with https://github.com/robotology/human-dynamics-estimation/pull/223/commits/3add0d577519d2594cfe4073b5aaaefdc2fb0c2f I add the possibility to pass this option simply as a string, and I have added a deprecation warning when it is passed as list (even if the list is still supported in order to avoid breaking old configuration files).  I have also changed the name of the variable `floatingBaseFrameList` to `baseFrameName` in order t be more clean.

Reply:
I think the choice of using `floatingBaseFrame` as a list https://github.com/robotology/human-dynamics-estimation/blob/devel/conf/xml/Human.xml#L23 was based on the choice of link and joint map information parameters we used https://github.com/robotology/human-dynamics-estimation/blob/devel/conf/xml/Human.xml#L55-L79

Thanks for updating it and keeping back compatibility.